### DISCUSSION TITLE
v1/chat/completions 接口报 502

### DISCUSSION THREAD

**Discussion by cantyCai** [chatchat-space/Langchain-Chatchat#4084] [Q&A] [answered]
server_config.py
# fastchat openai_api server
FSCHAT_OPENAI_API = {
    "host": DEFAULT_BIND_HOST,
    "port": 20006,
}

模型LLM主要通过接口方式访问，LLM_MODELS = ["zhipu-api", "openai-api"]，两个模型都配置了key，openai配置了请求地址，liunx环境是好的。
本地有代理，在本地调试，通过执行search_knowledgebase_simple主函数进行调试，最后到llm模型接口调用的时候报错，报INFO: HTTP Request: POST http://127.0.0.1:20006/v1/chat/completions "HTTP/1.1 502 Bad Gateway"，是什么原因？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
这个原因有点多，很有可能是模型这边没通，在现在0.3.x已经解耦了，可以检测模型测这是否返回内容

**Reply by Amos20191021** [reply]
这个就是我们部署时候拉源使用了科学上网：我是Windows机器用梯子，然后Linux服务器全局代理向这台Windows。等到跑起来的时候，这个全局代理未取消，而导致，unset http_proxy
unset https_proxy
就可以了
